source/easyleed/defaultconfig.py

```diff
@@ -1,84 +1,177 @@
+        # change in position per key press if Ctrl pressed
+        self.QGraphicsMovableItem_smallMove = eval(self.GUIConfig['QGraphicsMovableItem_smallMove'])
 
-######################
-######################
+        ## QGraphicsSpotItem ##
+        # change in radius of the spot per key press (+/-) in pixel
+        self.QGraphicsSpotItem_spotSizeChange = eval(self.GUIConfig['QGraphicsSpotItem_spotSizeChange'])
 
-#### Tracking related ####
-##########################
+        ## QGraphicsCenterItem ##
+        self.QGraphicsCenterItem_size = eval(self.GUIConfig['QGraphicsCenterItem_size'])
 
-# precision of the user input (standard deviation in pixel)
-Tracking_inputPrecision = 2
-# scale the integration window with changing energy
-Tracking_windowScalingOn = True
-# minimal radius of the integration window (in pixel)
-Tracking_minWindowSize = 0
-# function for spot identification
-Tracking_guessFunc = "guess_from_Gaussian"
-# Kalman tracker process noise
-Tracking_processNoisePosition = 0.1
-Tracking_processNoiseVelocity = 0.0
-# size of validation region
-# Ideal assumptions D_M^2 ~ Chi^2 with two degrees of freedom
-# cdf Chi^2 with two degrees of freedom is 1 - exp(-x/2)
-Tracking_gamma = 8
-# Minimal coefficient of determination R^2 for fit
-Tracking_minRsq = 0.8
-# factor by which the fitted region is bigger than the radius
-Tracking_fitRegionFactor = 1.5
+        ##########################
+        #### Tracking related ####
+        ##########################
 
-##########################
-##########################
+        # precision of the user input (standard deviation in pixel)
+        self.Tracking_inputPrecision = eval(self.trackingConfig['Tracking_inputPrecision'])
+        # scale the integration window with changing energy
+        self.Tracking_windowScalingOn = eval(self.trackingConfig['Tracking_windowScalingOn'])
+        # minimal radius of the integration window (in pixel)
+        self.Tracking_minWindowSize = eval(self.trackingConfig['Tracking_minWindowSize'])
+        # function for spot identification
+        self.Tracking_guessFunc = self.trackingConfig['Tracking_guessFunc']
+        # Kalman tracker process noise
+        self.Tracking_processNoisePosition = eval(self.trackingConfig['Tracking_processNoisePosition'])
+        self.Tracking_processNoiseVelocity = eval(self.trackingConfig['Tracking_processNoiseVelocity'])
+        # size of validation region
+        # Ideal assumptions D_M^2 ~ Chi^2 with two degrees of freedom
+        # cdf Chi^2 with two degrees of freedom is 1 - exp(-x/2)
+        self.Tracking_gamma = eval(self.trackingConfig['Tracking_gamma'])
+        # Minimal coefficient of determination R^2 for fit
+        self.Tracking_minRsq = eval(self.trackingConfig['Tracking_minRsq'])
+        # factor by which the fitted region is bigger than the radius
+        self.Tracking_fitRegionFactor = eval(self.trackingConfig['Tracking_fitRegionFactor'])
```

Comment:
Seems like it might be more elegant (and safer than eval) to use the getfloat(), getboolean() methods of configparser. Additionally you would also get some level of sanity check on inputs, as these methods throw an error if the conversion fails, which eval does not.